docs/api.md

```diff
@@ -658,6 +697,62 @@ If the OAuth process failed, the redirect will have `status=failed` appended to
     303 See Other
     Location: (redirect URI with status=failed)
 
+#### Get OAuth accounts info
+
+Visibility: Public
+
+`GET /oauth/info`
+
+Returns relevant oauth information for the current session.
+
+#### Success:
+
+    200 Ok
+
+    {
+      "result": [
+        {
+          "provider": "google"|"apple",
+          "provider_account_id": "91293",
+          "email": "authn@keratin.com"
+        }
+      ]
+    }
+
+#### Failure:
+
+    401 Unauthorized
+
+#### Delete OAuth account
+
+Visibility: Public
+
+`DELETE /oauth/:providerName`
+
+| Params | Type | Notes |
+| ------ | ---- | ----- |
+| `providerName` | string | * google |
+
+Delete an OAuth account from the current session. If the session was initiated via the OAuth flow, the account will be flagged to reset the password during the next sign-in attempt.
```

Comment:
Is this language right?  It looked to me like the removal would be blocked if a password change was required but maybe I misread.  I hope we wouldn't reset the password the next time someone tried to login based on just the emaiil.

Reply:
Have improved the doc here